ecosystem/resources/members/qiskit-qrack-provider.toml

```
@@ -0,0 +1,14 @@
+name = "Qiskit Qrack Provider"
+url = "https://github.com/vm6502q/qiskit-qrack-provider"
+description = "Qrack is a GPU-accelerated simulator with optional pure-CPU and CPU/GPU-hybrid simulation, scaling from minimalist systems to multiple GPUs for high performance, with novel near-Clifford, Schmidt-decomposition, quantum binary decision diagram (QBDD), light-cone optimization, state vector, and other simulation techniques."
```

This description needs to be <= 135 characters. I've had a go at rephrasing, what do you think of this?

```suggestion
description = "Supports near-Clifford, Schmidt-decomposition, quantum binary decision diagrams, and more. Scales from minimal setups to multiple GPUs."
```

---

Works for me!